src/components/DetailsMetrics/DetailsMetrics.js

```diff
@@ -122,13 +188,61 @@ const DetailsMetrics = ({ selectedItem }) => {
     )
   }, [dispatch, selectedItem])
 
+  useEffect(() => {
+    const selectedDate = detailsStore.dates.selectedOptionId
+    if (!selectedDate || !(selectedDate in timeRangeMapping)) return
+
+    setSelectedDate(timeRangeMapping[selectedDate])
+  }, [detailsStore.dates.selectedOptionId])
+
+  const fetchData = useCallback(
+    (params, params2, selectedItem) => {
```

Comment:
params and params2 are not names of parameters that should be used

Reply:
renamed:

- selectedMetricsParams
- preInvocationMetricParams